pyOCD/coresight/ap.py

```diff
@@ -337,8 +337,10 @@ def _handle_error(self, error, num):
 
 class AHB_AP(MEM_AP):
     def init(self, bus_accessible=True):
-        super(AHB_AP, self).init(bus_accessible)
+        # Init super first with bus access disabled, so it doesn't read the ROM table.
+        super(AHB_AP, self).init(False)
```

Comment:
Ah, you're right. Don't know why I thought ROM tables were optional on all APs. I don't have time to change this now, but I'll add it to my todo list.